source/libraries/Connector.ts

```diff
@@ -1,7 +1,7 @@
 import EthjsHttpProvider = require('ethjs-provider-http');
 import EthjsQuery = require('ethjs-query');
 import { TransactionReceipt } from 'ethjs-shared';
-import { Configuration } from './Configuration';
+import { Configuration } from '../libraries/Configuration';
```

Comment:
Same comment as above, and I see it in several other places below (this will be the last comment on the topic).